.github/workflows/continuous-integration.yml

```diff
@@ -156,3 +206,334 @@ jobs:
 
       - name: "Upload Code Coverage"
         uses: "codecov/codecov-action@v1"
+
+  phpunit-postgres:
+    name: "PHPUnit with PostgreSQL"
+    runs-on: "ubuntu-latest"
+    needs: phpunit-sqlite
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.4"
+        postgres-version:
+          - "9.3"
+          - "9.4"
+          - "9.5"
+          - "9.6"
+          - "10"
+          - "11.1"
```

Comment:
11.1 is very old. just specify `11` for the latest?

Reply:
add 12?

Reply:
add 13?

Reply:
That was introduced in 0a165b465ddbc4312be469e8a6b762877a61c87a , I don't know the reason for that but maybe @morozov recalls?

Reply:
I also think we could drop 9.2 (I can't get it to work but IIRC it's not supported anymore). @morozov , what should the final list look like?

Reply:
There is code in DBAL for PostgreSQL 9.2 which is technically still supported by DBAL 2.x and is continuously tested on Travis. If for some reason we can't test it on GitHub Actions, can we leave this build on Travis? We can get rid of it as soon as we stop supporting 2.x or manage to migrate to GitHub Actions.

Reply:
Why don't you add the failing PostgreSQL 9.2 build to the PR to possibly have another pair of eyes on it?

Reply:
> Why don't you add the failing PostgreSQL 9.2 build to the PR to possibly have another pair of eyes on it?

Sure, done, please take a look 👍

If we can't find a solution, then for sure we can leave the build on Travis.